sopel/config/core_section.py

```diff
@@ -389,57 +564,86 @@ def homedir(self):
 
     :default: ``chat.freenode.net``
 
-    **Required.**
+    **Required**:
+
+    .. code-block:: ini
+
+        host = chat.freenode.net
+
     """
 
     host_blocks = ListAttribute('host_blocks')
     """A list of hostnames which Sopel should ignore.
 
-    Messages from any user whose connection hostname matches one of these values
-    will be ignored. :ref:`Regular expression syntax <re-syntax>` is supported.
+    Messages from any user whose connection hostname matches one of these
+    values will be ignored. :ref:`Regular expression syntax <re-syntax>` is
+    supported:
+
+    .. code-block:: ini
+
+        host_blocks =
+            domain\\.com
+            .*\\.domain\\.com
 
     Also see the :attr:`nick_blocks` list.
     """
 
     log_raw = ValidatedAttribute('log_raw', bool, default=False)
     """Whether a log of raw lines as sent and received should be kept.
 
-    See :ref:`Raw Logs`.
+    To enable this logging:
+
+    .. code-block:: ini
+
+        log_raw = yes
```

Comment:
```suggestion
        log_raw = True
```

I'm partial to using the Pythonic form of a Boolean for these, but you do you. 😹

Reply:
As discussed on IRC, I don't want to give the impression that it's a Python value (like it would be evaluated as such), when it's not.

I'll also add a small section to explain that you can use more than "yes", but also "y", "on", etc. for any boolean value.